content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -221,3 +263,21 @@ To provide domain verification:
 If your validator's master private key is compromised, you must revoke it immediately and permanently.
 
 For information about how to revoke a master key pair you generated for your validator using the `validator-keys` tool, see [Key Revocation](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md#key-revocation).
+
+
+
+## Verify Your Connection to Network
+
+Here are some methods you can use to verify that your validator has a healthy connection to the XRP Ledger network:
+
+- Use the [`peers`](peers.html) command to return a list of all `rippled` servers currently connected to your validator. If the `peers` array is `null`, you don’t have a healthy connection to the network. If you've set up your validator using the instructions in this document, the `peers` array should include the same number of objects as the number of peers defined in your `[ips_fixed]` stanza. ***TODO: I have both r.ripple.com 51235 and zaphod.alloy.ee 51235 in my [ips_fixed] - and sometimes I get connect to only 1 peer. Is this a problem, or is it okay? If this is okay, I guess we can't say that the number of peers in ips_fixed and the number of peers returned will always be the same. If you are not using [peer_private] - what is a healthy minimum number of peers?***
```

Comment:
It should be the same. You can also check server_info for the 'peers' number to check this. With peer_private (which is now default for validators), this means you should only have 2 peers if you use the default.

Reply:
I think if you listed public hubs and they're busy, they may reject you, which is how you'd end up with fewer connections than your configured `[ips_fixed]`. That's not necessarily a problem on your end, so these instructions should account for that.